src/main/java/com/bina/varsim/util/Position.java

```diff
@@ -0,0 +1,52 @@
+package com.bina.varsim.util;
+
+/**
+ * base class for base-0 or base-1 location
+ */
+public abstract class Position extends Number{
```

Comment:
These values are unsigned... you may want to add some checks to avoid weirdness in the future. Don't worry about wasting half the range.